src/metabase/query_processor/middleware/annotate.clj

```diff
@@ -114,13 +114,14 @@
 (defn- display-name-for-joined-field
   "Return an appropriate display name for a joined field. For *explicitly* joined Fields, the qualifier is the join
   alias; for implicitly joined fields, it is the display name of the foreign key used to create the join."
-  [field-display-name {:keys [fk-field-id], join-alias :alias}]
+  [field-display-name {:keys [fk-field-id], join-alias :alias, original-join-alias :alias/original}]
```

Comment:
I don't understand why this is `:alias/original` and not `:join-alias/original`

Reply:
The key inside the join itself is `:alias` e.g.

```clj
{:joins [{:source-table 1, :alias "my_join", :condition [:= ...]}]}
```

so I just copied that. `:join-alias` would be redundant I guess